### ISSUE TITLE
Wrong file

### ISSUE THREAD

**Issue by Mylloon** [niksingh710/basic-battery-stat#1]
On my computer, the file aren't `energy_` but `charge_`

Error:
```
/usr/bin/cat: /sys/class/power_supply/BAT0/energy_full: Aucun fichier ou dossier de ce type
/usr/bin/cat: /sys/class/power_supply/BAT0/energy_full_design: Aucun fichier ou dossier de ce type
/usr/bin/basic-battery-stat: ligne 12: * 100 /  : erreur de syntaxe : opérande attendu (le symbole erroné est « * 100 /  »)
```

Using `charge` solves the issue

**Comment by semi710** [comment]
@Mylloon  Thanks for informing regarding that.
would generalise that for this exception too.
can you please share the output of 
```
/usr/bin/ls /sys/class/power_supply/BAT0/
```
just make sure to be root user while using the command.

**Comment by Mylloon** [comment]
```
alarm		charge_full_design  device	  power		 subsystem   voltage_min_design
capacity	charge_now	    hwmon2	  present	 technology  voltage_now
capacity_level	current_now	    manufacturer  serial_number  type
charge_full	cycle_count	    model_name	  status	 uevent
```

**Comment by semi710** [comment]
I am quite confused that I should add a check for the files between `charge_full` and `energy_full` or should use a regex to match `*_full`

**Comment by semi710** [comment]
should be fixed in ba7aff547207412969e6e2a9ec2af2ddd8fa4fdd check after updating and tell.

**Comment by Mylloon** [comment]
Working, thanks!
